docker-compose.yml

```diff
@@ -0,0 +1,113 @@
+version: "3.7"
+
+services:
+  sites-public:
+    container_name: sites-public
+    build:
+      context: .
+      dockerfile: Dockerfile.sites-public
+      target: development
+    volumes:
+      - ./sites/public:/usr/src/app/sites/public
+      - /usr/src/app/node_modules
+      - /usr/src/app/sites/public/node_modules
+    ports:
+      # TODO: configure via .env separate from ./sites/public/.env
+      - "3000:3000"
+    env_file:
+      - ./sites/public/.env
+    environment:
+      # TODO: login currently doesn't work, likely needs a separate env variable to
```

Comment:
Can you make an issue for this?

Reply:
That's CityOfDetroit/affordable-housing-app#84. I'll add that number in the comment.